backend/src/main/java/gov/cdc/usds/simplereport/db/model/DeviceTestPerformedLoincCode.java

```diff
@@ -15,10 +16,9 @@
 @Getter
 public class DeviceTestPerformedLoincCode extends IdentifiedEntity {
 
-  @ManyToOne
   @JoinColumn(name = "device_type_id")
   @JsonIgnore
-  private DeviceType deviceType;
+  private UUID deviceTypeId;
```

Comment:
I believe this can just be `@Column(name = "device_type_id")`
and `@JsonIgnore` is needed anymore